pkgs/tools/misc/mons/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, lib, fetchFromGitHub, fetchpatch, help2man, xrandr }:
+
+stdenv.mkDerivation rec {
+    pname = "mons";
+    version = "20200107";
+
+    src = fetchFromGitHub {
+        owner = "Ventto";
+        repo = pname;
+        rev = "0c9e1a1dddff23a0525ed8e4ec9af8f9dd8cad4c";
+        sha256 = "02c41mw3g1mgl91hhpz1n45iaqk9s7mdk1ixm8yv6sv17hy8rr4w";
+        fetchSubmodules = true;
+    };
+
+    preConfigure = ''sed -i 's/usr\///' Makefile'';
```

Comment:
I recommend making a PR or issue upstream so that they use `$(DESTDIR)$(PREFIX)` in the makefile, where PREFIX defaults to `/usr`.